x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_details_flyout/accordion_list.tsx

```diff
@@ -19,22 +19,28 @@ import {
   useGeneratedHtmlId,
 } from '@elastic/eui';
 
+import './accordion_list.scss';
+
 interface Props {
   iconType: IconType;
   items: string[];
+  hasBorder?: boolean;
+  initialIsOpen?: boolean;
```

Comment:
* `hasBorder` allows the accordion to be rendered with or without a border and padding. I've got some designs in the works that will use be making use of this component in both scenarios.
* `initialIsOpen` allows the accordion to be expanded. In this case specifically, we set the prop to `true` when items exist in the list.